databuilder/databuilder/extractor/dashboard/apache_superset/__init__.py

```diff
@@ -0,0 +1,164 @@
+import abc
+from functools import reduce
+from typing import (
+    Any, Dict, Iterator, List, Tuple,
+)
+
+import requests
+from dateutil import parser
+from pyhocon import ConfigFactory, ConfigTree
+
+from databuilder.extractor.base_extractor import Extractor
+
+type_fields_mapping = List[Tuple[str, str, Any, Any]]
+
+
+class ApacheSupersetBaseExtractor(Extractor):
```

Comment:
also could we put the extractor as a separate file (e.g https://github.com/amundsen-io/amundsen/blob/main/databuilder/databuilder/extractor/dashboard/mode_analytics/mode_dashboard_extractor.py)?